### PR TITLE
Cleanup quote links & photos

### PR DESCRIPTION
**Pull request by mirisuzanne**
## Cute animal pic
_Because everyone likes pictures of [animals](https://unsplash.com/t/animals)._

## Link to Trello card (if applicable)
_If this PR relates to a Trello card, don't forget to reference this PR on the card as well._

https://trello.com/c/Q9Xt2pvl/89-what-to-do-about-logo-permissions-and-photo-permissions

## Description
_The commit messages say what you did; this should explain why and/or how._
- [x] Description is in Trello card

– If the quote is pulled from a public link, and there is a professional-looking author photo available at the link (eg twitter avatar, or an article byline), then we should go ahead and use that same photo.
- If a twitter avatar is more personal, we just the twitter icon
- If the quotes were given to us privately, like client quotes, we ask permission for them
- Link to twitter/article when quotes are taken from public links

## Steps to test/reproduce
_Please explain how to best reproduce the issue and/or test the changes locally (including the pages/URLs/views/states to review)._

Quotes are usually scattered across various products & talks… so I created a sample page with all of them at `/sample/press/`

Screenshot_2020-07-14 All Press Quotes

### CODE REVIEW

**Comment by mirisuzanne** [comment]
[What to do about logo permissions and photo permissions?](https://trello.com/c/Q9Xt2pvl/89-what-to-do-about-logo-permissions-and-photo-permissions)

File: content/talks/code-reviews.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ events:
     date: 2015-11-11
     slides: https://www.slideshare.net/Funstacy/sass-code-reviews-how-one-code-review-changed-my-life-sassconf2015
   - venue: CSS Dev Conf
-    url: https://twitter.com/cssdevconf
+    url: http://2015.cssdevconf.com/
```
Comment (jgerigmeyer) [inline content/talks/code-reviews.md:19]: @mirisuzanne Why this change? The link is broken for me, which is why I replaced it with the Twitter link.

Comment (mirisuzanne) [reply content/talks/code-reviews.md:19]: It works fine for me, so I'm confused.

Comment (jgerigmeyer) [reply content/talks/code-reviews.md:19]: Weird. Apparently my FF has a cached redirect to https:// that I can't get rid of... it works in Chrome 👍
